src/mixins/extensions/ValidationRules.js

```diff
@@ -10,8 +11,14 @@ export default {
           delete properties[':validation'];
           delete properties['validation'];
           // Add validation class and error message
-          properties[':class'] = `{ 'form-group--error': ${this.checkVariableExists('$v.vdata.' + element.config.name)} && $v.vdata.${element.config.name}.$invalid || ${this.checkVariableExists('$v.schema.' + element.config.name)} && $v.schema.${element.config.name}.$invalid }`;
-          properties[':error'] = `${this.checkVariableExists('$v.vdata.' + element.config.name)} && validationMessage($v.vdata.${element.config.name}) || ${this.checkVariableExists('$v.schema.' + element.config.name)} && validationMessage($v.schema.${element.config.name})`;
+          if (!this.isValidVariableName(element.config.name)) {
+            if (this.$store.getters["globalErrorsModule/getMode"] === "preview") {
```

Comment:
To improve the reading of this code, you could try to map Getters.
see: https://blog.logrocket.com/best-practices-mapping-vuex/#whatarevuexmapgetters